features/conference/index.js

```diff
@@ -1 +1,2 @@
 export * from './components';
+export { default as styles } from './styles';
```

Comment:
A very good point, @tsareg!

Even if there was a need, we haven't allowed it so it's a no-go. @legastero, please don't modify this PR though because I'm moving the styles directory back in components to see how it goes.